README.md

```diff
@@ -1 +1,31 @@
-# EjercicioPoo3
\ No newline at end of file
+# EjercicioPoo3
+
+ejercicio una calculadora que me sume,reste, multiplique, potencie y calcule raiz de manera que se le digiten los numeros o de forma random.
+
+en este ejercio se refuerza conceptos de:
+
+-metodos
+-constructores
+-return
+-objeto(clase)
+-instancias
+
+autor
+
+Esteban Vanegas A.
+
+------------------------------------
+
+I exercise a calculator that adds, subtracts, multiplies, potentiates and calculates the root so that the numbers are entered or randomly.
```

Comment:
Please rename this

This exercise is a calculator that..... root and the numbers are entered or generated randomly